cinema_service/urls.py

```diff
@@ -1,3 +1,5 @@
+from django.conf import settings
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
```

Comment:
Consider grouping the imports using parentheses to improve code readability, as per the checklist recommendation.